include/jwt-cpp/jwt.h

```diff
@@ -688,12 +731,12 @@ namespace jwt {
 		return verifier<Clock>(c);
 	}
 
-	auto verify() {
-		struct default_clock {
-			auto now() const {
-				return std::chrono::system_clock::now();
-			}
-		};
+	struct default_clock {
+		std::chrono::_V2::system_clock::time_point now() const {
```

Comment:
Shouldn't this be std::chrono::system_clock::time_point ?

Otherwise thank you for your contribution, will merge as soon as possible.

Reply:
Fixed. And thank you for this lightweight library